draft-iab-path-signals-collaboration.md

```diff
@@ -505,7 +506,10 @@ undesirable.
 
 This document recommends that signalling mechanisms that send information
 are built to specifically support sending the necessary, minimal set of information (see {{minimize-info}})
-and no more. Such mechanisms also need have an ability for establishing an agreement (see {{control-distr}}) and to establish
+and no more. As previsouly noted flow-identifying information are a path signal in itself,
+and as such provisioning of flow identifiers also requires protocol specific analysis.
```

Comment:
What do you not understand? It is mainly to answer the question in the issue that there is not one common way how to identify a flow.

Reply:
"provisioning of flow identifiers also requires protocol specific analysis"... it seems a bit vague to just say that we need analysis. The previous sentence is about saying that we need to minimize data.

Shouldn't we be saying that "since flow identifiers are signals too, even they need to be minimized in how much they are reused", or something similar?

Reply:
why reused? The part of minimisation is probably covered in the earlier section. This one is more to say that it probably not always right to use the IP address (even if it often just there) but also design/select the identifiers based on the protocol used and function needed.

I agree my text might be vague but I also didn't want to add something length... or should we write more?